data/map.txt

```diff
@@ -1973,7 +1973,7 @@ system Acrux
 		period 11.4474
 		offset 180
 	object
-		sprite planet/dust5
+		sprite "planet/gas2"
```

Comment:
Should this have `-hot`?
You also don't need quotes around any of these anymore since the sprite name has no spaces.